contracts/2022/06/Proposal27.sol

```diff
@@ -4,14 +4,6 @@ pragma solidity 0.8.10;
 import {YAMTokenInterface} from "../../../utils/YAMTokenInterface.sol";
 import {IERC20} from "openzeppelin-contracts/token/ERC20/IERC20.sol";
 
-interface IYVault {
-    function deposit(uint256 amount, address recipient)
-        external
-        returns (uint256);
-
-    function withdraw(uint256 amount) external returns (uint256);
-}
-
```

Comment:
that breaks the contract accounting for whats on [`line 13-14`](https://github.com/yam-finance/proposals/blob/6df5c2caf72aeef05994e8b0c0ab5abd469142b9/contracts/2022/06/Proposal27.sol#L13-L14)

Reply:
correct